src/client/utils/styles/_variables.scss

```diff
@@ -69,7 +69,7 @@ $icon-hover:              #888;
 
 // other
 
-$hover:                   lighten($brand, 43);
+$hover:                   hsl(0, 0%, 73%);
```

Comment:
this is too dark I think, I can't be sure from the attached screenshots but I think the color for hover should be the same as for `item-measure`: #e0e4ef